torchvision/csrc/ops/cuda/deform_conv2d_kernel.cu

```diff
@@ -372,6 +372,7 @@ __global__ void deformable_col2im_kernel(
 
     const scalar_t y = (out_y * stride_h - pad_h) + i * dilation_h + offset_h;
     const scalar_t x = (out_x * stride_w - pad_w) + j * dilation_w + offset_w;
+    const index_t grad_im_numel = width * height * channels * batch_sz;
```

Comment:
This doesn't need to be in the loop right

Reply:
Oh, right, I haven't remarked `CUDA_1D_KERNEL_LOOP_T`, thanks! Fixed.